frontend/src/components/auth/Login.jsx

```diff
@@ -97,21 +100,48 @@ const Login = () => {
         navigate('/signup')
     }
 
+    const handleHome = ()=>{
+        navigate('/')
+    }
+
+    
+
     return (
-        <div className='login-form'>
-            <h2>Login</h2>
-            <form onSubmit={handleLogin}>
-                <input type='email' placeholder='Email' value={email} onChange={(e) => setEmail(e.target.value)} required/>
-                <input type='password' placeholder='Password' value={password} onChange={(e) => setPassword(e.target.value)} required/>
-                <button type='submit'>Login</button>
-            </form>
-            {error && <p>{errorMessage}</p>}
-            {/* TODO: finish up meta developers authentication. Must file business authentication. (takes minimum 5 days) */}
-            {/* <button onClick={handleFacebookLogin}>Login with Facebook</button> */}
-            <button className= "google-button" onClick={handleGoogleLogin}><i className="fa-brands fa-google"></i>Login with Google</button>
-            {error && <p>{googleErrorMessage}</p>}
-            <p className='form-footer'>Don't have an account? <button onClick={handleCreateAccount}>Signup</button></p>
-        </div>
+        <>
+            <nav className='navBar-container'>
+                <div className="navbar">
+                    <div className="brand">
+                        InterviewMe
+                    </div>
+                    <ul className="nav-links">
+                        <li><a type='button' onClick={handleHome} >Home</a></li>
+                        
+                    </ul>
+                </div>
+            </nav>
+            <Spacing/>
+            <div className='form-container'>
+                <div className='login-form'>
+                    <h2>Login</h2>
+                    <form onSubmit={handleLogin}>
+                        <input type='email' placeholder='Email' value={email} onChange={(e) => setEmail(e.target.value)} required />
+                        <input type='password' placeholder='Password' value={password} onChange={(e) => setPassword(e.target.value)} required />
+                        <button type='submit' className='login-button'>Login</button>
+                    </form>
+                    {error && <p className='error-message'>{errorMessage}</p>}
+                    {/* TODO: finish up meta developers authentication. Must file business authentication. (takes minimum 5 days) */}
+                    {/* <button onClick={handleFacebookLogin}>Login with Facebook</button> */}
```

Comment:
Just re-commenting here in case you didn't see my comment on the other diff that was already merged.

I recommend postponing FB login for now unless it is super simple to implement. Don't spend too much time on it.